### DISCUSSION TITLE
How to change DataGrid ColumnOptions filter icon?

### DISCUSSION THREAD

**Discussion by aliastopan** [microsoft/fluentui-blazor#1515] [Q&A] [answered]
In [customcomparer](https://www.fluentui-blazor.net/DataGrid#customcomparer) example, `TemplateColumn` could have `ColumnOptions`. This useful for adding column based filtering. But is it possible to change the default filter icon?

**Comment by vnbaaij** [comment, marked as answer]
You can change that icon by using the `HeaderCellItemTemplate` but when using that, it is left up to that template to render any relevant
"show options" UI and invoke the grid's `FluentDataGrid<TGridItem>.ShowColumnOptionsAsync(ColumnBase<{TGridItem>>` .

The code in `RenderDefaultHeaderContent` in `ColumnBase.razor` can be used as a guideline.
